src/Umbraco.Web.UI.Client/src/views/membertypes/edit.controller.js

```diff
@@ -31,21 +32,22 @@
 
         vm.page.keyboardShortcutsOverview = [
 			{
+                "name": localizationService.localize("shortcuts_shortcut").then(function (value) {return value;}),
 			    "shortcuts": [
 					{
-					    "description": "Add tab",
+					    "description": localizationService.localize("shortcuts_addTab").then(function (value) {return value;}),
 					    "keys": [{ "key": "alt" }, { "key": "shift" }, { "key": "t" }]
 					},
 					{
-					    "description": "Add property",
+					    "description": localizationService.localize("shortcuts_addProperty").then(function (value) {return value;}),
 					    "keys": [{ "key": "alt" }, { "key": "shift" }, { "key": "p" }]
 					},
 					{
-					    "description": "Add editor",
+					    "description": localizationService.localize("shortcuts_addEditor").then(function (value) {return value;}),
```

Comment:
Is this syntax required? Can't we just return the promise like:

"description": localizationService.localize("shortcuts_addEditor")
? 

@madsrasmussen or @clausjensen ?